tool/python/README.md

```diff
@@ -0,0 +1,191 @@
+    SINGAROOT/tool
```

Comment:
chonho's code is still included..
the config helper is not complete, hence cannot merge it together with this pull request.